### PR TITLE
DataBase is done

### PR DESCRIPTION
**Pull request by qAziz060708**

### CODE REVIEW

File: OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/DataBaseService.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        private readonly IDataBaseRepository _dataBaseRepository;
+
+        private readonly IMapper _mapper;
+
+        public DataBaseService(IDataBaseRepository dataBaseRepository, IMapper mapper)
+        {
+            _dataBaseRepository = dataBaseRepository;
+            _mapper = mapper;
+        }
+        public async Task<int> AddDataBaseAsync(DataBase dataBase)
```
Comment (isfandiyoro5) [inline OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/DataBaseService.cs:21]: line tashash kk

File: OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/DataBaseService.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+using PharmacyShopping.DataAccess.Models;
+using PharmacyShopping.DataAccess.Repository.IRepositories;
+using Microsoft.EntityFrameworkCore;
+
+namespace PharmacyShopping.BusinessLogic.Service.Services
+{
+    public class DataBaseService : IDataBaseService
+    {
+        private readonly IDataBaseRepository _dataBaseRepository;
+
```
Comment (isfandiyoro5) [inline OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/DataBaseService.cs]: bu lineni delete qilsa ham boladi

**Review by isfandiyoro5 — CHANGES_REQUESTED** [review]

File: OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/DataBaseService.cs (hunk 1 above)
Comment (qAziz060708) [reply OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/DataBaseService.cs:21]: qilindi

File: OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/DataBaseService.cs (hunk 2 above)
Comment (qAziz060708) [reply OnlinePharmacyShopping/PharmacyShopping.BusinessLogic/Service/Services/DataBaseService.cs]: Qilindi

**Pull request merged**